libr/anal/fcn.c

```diff
@@ -839,8 +839,13 @@ static int fcn_recurse(RAnal *anal, RAnalFunction *fcn, ut64 addr, ut8 *buf, ut6
 			{
 				bool must_eob = anal->opt.eobjmp;
 				if (!must_eob) {
-					RIOSection *s = anal->iob.section_vget (anal->iob.io, addr);
-					RIOSection *d = anal->iob.section_vget (anal->iob.io, op.jump);
+					SdbList *secs = anal->iob.section_vget_secs_at (anal->iob.io, addr);
```

Comment:
May be wrap this in a macro? `#define IO_SECS(arg, addr) (arg).section_vget_secs_at((arg).io, addr)` or even shorter, with `addr` is only as an argument.